README.md

```diff
@@ -253,6 +253,29 @@ export default class Hello extends Component {
 }
 ```
 
+Template tag components can also be used for writing tests. In fact, this aligned syntax between app code and test code is one of the big advantages of the new authoring format.
+
+Just like in app code, the template tag has access to the outer scope. This means you can reference variables directly in your tests:
```

Comment:
should we mention that there is now no need to interact with `this` outside of getting `this.owner`? :sweat_smile:

Reply:
Note sure if we want to go deeper into details in the README? The way I see it is more for the "hey look, this is possible" kind of things. Ideally we'd just have a whole section on testing in the official guides, once we get there.

But LMK if you see it different.

---

For which use case would you use `this.owner` in tests?

Reply:
> But LMK if you see it different.

naw, I think it's fine!

> For which use case would you use this.owner in tests?

only interacting with services directly